pandas-iris/{{ cookiecutter.repo_name }}/conf/base/parameters.yml

```diff
@@ -1,6 +1,4 @@
-# Parameters for the example pipeline. Feel free to delete these once you
-# remove the example pipeline from hooks.py and the example nodes in
-# `src/pipelines/`
-example_test_data_ratio: 0.2
-example_num_train_iter: 10000
-example_learning_rate: 0.01
+model_options:
+  train_fraction: 0.8
+  random_state: 3
+  target_column: species
```

Comment:
Please can we make these top-level parameters (i.e. outside `model_options`)? You'll need to where they are specified as node input accordingly.